logger/logger.go

```diff
@@ -416,15 +416,28 @@ func HTTPLogger(inner http.Handler, name string) http.Handler {
 	})
 }
 
+// IsSensitive checks if the given key exists in the list of bad words (sensitive info)
+func IsSensitive(key string) bool {
```

Comment:
Are we missing substring comparison?

Reply:
I'll update the PR to perform the substring comparison.